public/components/datasources/components/manage/accelerations/create_accelerations_flyout/create/utils.tsx

```diff
@@ -47,11 +47,16 @@ export const validateReplicaCount = (replicaCount: number) => {
 
 export const validateRefreshInterval = (
   refreshType: AccelerationRefreshType,
-  refreshWindow: number
+  refreshWindow: number,
+  refreshInterval: string
 ) => {
-  return refreshType === 'autoInterval' && refreshWindow < 1
-    ? ['refresh window should be greater than 0']
-    : [];
+  if (refreshType === 'autoInterval' && refreshInterval === 'minute' && refreshWindow < 15)
```

Comment:
Is it possible to have decimal numbers? like 0.1 hour?

Reply:
we use parse int to refresh window doesn't take decimal inputs